### DISCUSSION TITLE
Disable JIT mode and purge option

### DISCUSSION THREAD

**Discussion by igorlopasovsky** [tailwindlabs/tailwindcss#5137] [Help]
Is it possible to completely disable JIT mode and purge option? Unfortunately, my frontend is connected to backend (WordPress Gutenberg editor) and JIT is obviously not working correctly. Safe list is not an option here as this is a quite complex project. I would rather link a full Tailwind CSS file. Unfortunately, when I disable JIT mode and remove purge option `npm run watch` stops working. What's the correct approach here? I thought that removing JIT and purge from config will compile the full CSS file but my way of thinking is probably not correct.

**Comment by simonswiss** [comment]
Hey!

What is your `watch` script doing in `package.json`?

You are correct, if you remove the purge option (or set it to `false`), it should compile all the utility classes (based on your config's `theme` and `variants`).

Are you able to share some relevant code (config file, package.json, tooling involved?

Not too familiar with Gutenberg, but are you able to write a little script that loops through all your Gutenberg blocks and render them in a text file? 

That's a quick and dirty way, but I understand you'd need to re-run the script every time you make changes, so probably not ideal.
